components/molecules/ContributorListTableRow/contributor-list-table-row.tsx

```diff
@@ -134,13 +163,11 @@ const ContributorListTableRow = ({
 
           <div className="flex items-center justify-between py-3 border-b">
             <div>Languages</div>
-            {contributorLanguageList.length > 0 ? (
+            {contributorLanguageList && (
               <p>
-                {contributorLanguageList[0]}
-                {contributorLanguageList.length > 1 ? `,+${contributorLanguageList.length - 1}` : ""}
+                {contributorLanguageList[0] && getLanguageAbbreviation(contributorLanguageList[0])}
+                {contributorLanguageList[1] && `, ${getLanguageAbbreviation(contributorLanguageList[1])}`}
```

Comment:
`contributorLanguageList ` could technically return only 1 item

<img width="362" alt="Screenshot 2023-11-23 at 2 04 47 PM" src="https://github.com/open-sauced/app/assets/67210629/1146932d-8b53-48f1-8977-d84bed902ed6">

That is why I added that additional check for `contributorLanguageList[1]` because in some cases that will come back `undefined`